app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car_list: list[Car]) -> float:
+        income = 0
+        for car in car_list:
+            if car.clean_mark < self.clean_power:
+                income += (car.comfort_class
+                           * (self.clean_power - car.clean_mark)
+                           * self.average_rating
+                           / self.distance_from_city_center)
+                self.wash_single_car(car)
```

Comment:
Ensure that `wash_single_car` is called only if the car's `clean_mark` is less than `clean_power`. This is already checked in the `if` condition, so this line is correct.